backend/coreapp/compilers.py

```diff
@@ -34,6 +44,9 @@ class Compiler:
     is_gcc: ClassVar[bool] = False
     is_ido: ClassVar[bool] = False
     is_mwcc: ClassVar[bool] = False
+    is_mac_compiler: ClassVar[bool] = False
```

Comment:
`_compiler` is redundant here imo, unless you did that because it could imply the compiler's host machine is mac rather than the target?

Reply:
I just realized this flag (and the special class) can disappear anyway once the other fields disappear